docs/spfx/office-addins-create.md

```diff
@@ -44,7 +44,7 @@ You will need to deploy the solution to tenant using the **tenant scoped deploym
 Before you can use the Office JavaScript SDK in your code, you'll need to include correct types for the solution. You can install latest types by adding `@types/office-js` package from the npm to your solution with following command:
 
 ```shell
-npm install @types/office-js
+npm install @types/office-js --save
```

Comment:
This isn't correct... because they are type declarations, they should go in `devDependencies`. Please update to use the `--save-dev` argument.

Reply:
@andrewconnell  - Could you please let me know how do know that npm install --save (or --save-dev, I'm not sure) @types/[type_package_name]

Reply:
Thanks @andrewconnell for the explanation.
Updated the change.